sorting_methods/main.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+
+from algorithms import *
+import random
+
+SIZE = 100
+
+def generate_random_array(size):
+    array = []
+    for x in range(size): array.append(random.randint(0, size))
+    return array
+
+def generate_ascending_array(size):
+    return [*range(size)]
+
+def generate_descending_array(size):
+    return [*range(size, 0, -1)]
+
+
+if __name__ == "__main__":
+    array = generate_random_array(SIZE)
+    print("b4:", array)
+    print("\ninsert:", insert.sort(array.copy()))
+    print("\nquick:", quick.sort(array.copy()))
+
+    array = generate_ascending_array(SIZE)
+    print("\nb4:", array)
+    print("\ninsert:", insert.sort(array.copy()))
+    print("\nquick:", quick.sort(array.copy()))
+
+    array = generate_descending_array(SIZE)
+    print("\nb4:", array)
+    print("\ninsert:", insert.sort(array.copy()))
+    print("\nquick:", quick.sort(array.copy()))
```

Comment:
newline dodaj na koncu

Reply:
nie wiem, dlaczego tak się to tutaj wyświetla, ale zarówno pliki u mnie na dysku, jak i te tutaj po pobraniu, mają znak nowej linii na końcu; mam nadzieję, że nie będzie to miało znaczenia przy kolejnych listach